conans/client/tools/oss.py

```diff
@@ -24,12 +25,24 @@ def args_to_string(args):
 
 def cpu_count(output=None):
     output = default_output(output, 'conans.client.tools.oss.cpu_count')
+
+    def get_cpus():
+        base_cpus = "/sys/fs/cgroup/cpu"
+        try:
+            cfs_quota_us = int(load(os.path.join(base_cpus, "cpu.cfs_quota_us")))
+            cfs_period_us = int(load(os.path.join(base_cpus, "cpu.cfs_period_us")))
+            if cfs_quota_us > -1 and cfs_period_us > 0:
```

Comment:
please change to `cfs_quota_us > 0`, otherwise we can get `-j0`. probably raise here (earlier), so error message is more clear, than just that `-j0` is incorrect from make

Reply:
done